test/test-pages/hukumusume/expected.html

```diff
@@ -67,23 +56,6 @@
                     </tbody>
                 </table>
             </div>
-            <table>
-                <tbody>
-                    <tr>
-                        <td> ♪音声配信(html5) </td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <audio src="http://ohanashi2.up.seesaa.net/mp3/ae_0101.mp3" controls=""></audio>
-                        </td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <a href="http://www.voiceblog.jp/onokuboaki/"><span size="-1">亜姫の朗読☆　イソップ童話より</span></a>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
```

Comment:
Hm, why do we now delete this when we did not before? Is this a result of your patch?